cmd/dp-observation-extractor/main.go

```diff
@@ -63,11 +63,11 @@ func main() {
 	checkForError(ctx, err)
 
 	// Kafka Observation Producer
-	kafkaObservationProducer, err := serviceList.GetProducer(ctx, config.KafkaAddr, config.ObservationProducerTopic, initialise.Observation)
+	kafkaObservationProducer, err := serviceList.GetProducer(ctx, config.KafkaAddr, config.ObservationProducerTopic, initialise.Observation, config)
```

Comment:
This is not too important, but a suggestion: we could prevent passing config.KafkaAddr, and instead, read it from config in `serviceList.GetProducer` straight away, so that we can reduce the number of parameters for this funciton.